airflow/providers/databricks/hooks/databricks.py

```diff
@@ -69,6 +69,10 @@
 
 SPARK_VERSIONS_ENDPOINT = ("GET", "api/2.0/clusters/spark-versions")
 
+CREATE_REPO_ENDPOINT = ("POST", "api/2.0/repos")
+DELETE_REPO_ENDPOINT = ("DELETE", "api/2.0/repos/{repo_id}")
+UPDATE_REPO_ENDPOINT = ("PATCH", "api/2.0/repos/{repo_id}")
```

Comment:
Why do you need to define them as tuple then create a lambda to transform them? You can just create a method/some methods to return these values.

Reply:
@hussein-awala personally I think it would be most correct to update `_self_do_api(...)` to take in two separate strings instead of tuple

Reply:
> Why do you need to define them as tuple then create a lambda to transform them? You can just create a method/some methods to return these values.

I just figured we wanted to maintain some consistency with the other constants, and felt like it might be confusing if we had something like `CREATE_REPO_ENDPOINT_HEADER` and `CREATE_REPO_ENDPOINT_ROUTE` which would clash. 

I don't think I fully understand your question though, are you suggesting we just have a function as opposed to lambda i.e. 
```python

def modify_route(route: tuple, repo_id: str) -> tuple:
    header = route[0]
    route = route[1].format(repo_id=repo_id)
    return (header, route)
```

Reply:
I too think we can simplify this by just adding method(s).